mmv1/products/cloudrunv2/api.yaml

```diff
@@ -0,0 +1,715 @@
+            - !ruby/object:Api::Type::String
+              name: "severity"
+              output: true
+              description: |-
+                How to interpret failures of this condition, one of Error, Warning, Info
+            - !ruby/object:Api::Type::String
+              name: "reason"
+              output: true
+              description: |-
+                A common (service-level) reason for this condition.
+            - !ruby/object:Api::Type::String
+              name: "revisionReason"
+              output: true
+              description: |-
+                A reason for the revision condition.
+            - !ruby/object:Api::Type::String
+              name: "executionReason"
+              output: true
+              description: |-
+                A reason for the execution condition.
+      - !ruby/object:Api::Type::String
+        name: "latestReadyRevision"
+        output: true
+        description: |
+          Name of the latest revision that is serving traffic. See comments in reconciling for additional information on reconciliation process in Cloud Run.
+      - !ruby/object:Api::Type::String
+        name: "latestCreatedRevision"
+        output: true
+        description: |
+          Name of the last created revision. See comments in reconciling for additional information on reconciliation process in Cloud Run.
+      - !ruby/object:Api::Type::Array
+        name: "trafficStatuses"
+        description: |-
+          Detailed status information for corresponding traffic targets. See comments in reconciling for additional information on reconciliation process in Cloud Run.
+        output: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: "type"
+              output: true
+              description: |-
+                The allocation type for this traffic target.
+            - !ruby/object:Api::Type::String
+              name: "revision"
+              output: true
+              description: |-
+                Revision to which this traffic is sent.
+            - !ruby/object:Api::Type::Integer
+              name: "percent"
+              output: true
+              description: |-
+                Specifies percent of the traffic to this Revision.
+            - !ruby/object:Api::Type::String
+              name: "tag"
+              output: true
+              description: |-
+                Indicates the string used in the URI to exclusively reference this target.
+            - !ruby/object:Api::Type::String
+              name: "uri"
+              output: true
+              description: |-
+                Displays the target URI.
+      - !ruby/object:Api::Type::String
+        name: "uri"
+        output: true
+        description: |
+          The main URI in which this Service is serving traffic.
+      - !ruby/object:Api::Type::Boolean
+        name: "reconciling"
```

Comment:
Should the resource wait until not reconciling before updating? similar to go/tpg-resource-state-handling

Reply:
I double checked and I think this should be handled by waiting on the operation. So maybe we can remove this field?

Reply:
I'm fine either way